erp/export/utils.py

```diff
@@ -0,0 +1,50 @@
+from abc import abstractmethod
+from dataclasses import dataclass
+from typing import List, Tuple, Type
+
+from erp.models import Erp
+
+
+@dataclass(frozen=True)
+class BaseExportMapper:
+    @staticmethod
+    @abstractmethod
+    def headers():
+        ...
```

Comment:
Nit: Why are we using Python Ellipsis here and elsewhere? Doesn't seem like a valid usage… https://docs.python.org/dev/library/constants.html#Ellipsis 

I'd rather use a standard `pass` statement